### DISCUSSION TITLE
ANY ONE can give a install method without Docker？

### DISCUSSION THREAD

**Discussion by lyksdu** [THUDM/AgentBench#138] [Q&A]
how to install the bench when I am not familiar with the docker, I plan to install all the things in a .bat/.sh. Is this possible for this repo？

**Comment by zhc7** [comment]
Hi, although it's possible theoratically, the amount of work and conflicts between task environments make it highly improbable. Some tasks have requirements under `src/server/tasks/<task_name>`, you may start from there.

**Comment by YerongLi** [comment]
> how to install the bench when I am not familiar with the docker, I plan to install all the things in a .bat/.sh. Is this possible for this repo？

Did you figure out how to install everything without docker?
